### PR TITLE
[ParticleMechanics] Modifying constitutive law import

### PR DESCRIPTION
**Pull request by bodhinandach**
Addressing #3791. Sorry for the late response @loumalouomega @philbucher

### CODE REVIEW

**Review by philbucher — APPROVED** [review]
Thanks

File: applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py [hunk 1]
```diff
@@ -364,10 +364,10 @@ def _execute_check_and_prepare(self):
     def _import_constitutive_laws(self):
         materials_filename = self.settings["material_import_settings"]["materials_filename"].GetString()
         if (materials_filename != ""):
-            import read_materials_process
             # Add constitutive laws and material properties from json file to model parts.
-            read_materials_process.ReadMaterialsProcess(self.model, self.settings["material_import_settings"])
-
+            material_settings = KratosMultiphysics.Parameters("""{"Parameters": {"materials_filename": ""}} """)
+            material_settings["Parameters"]["materials_filename"].SetString(materials_filename)
+            KratosMultiphysics.ReadMaterialsUtility(material_settings, self.model)
```
Comment (philbucher) [inline applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py:370]: @loumalouomega why did you not pass the `self.settings["material_import_settings"]` in StructuralMechanics?
@bodhinandach just copied this but I am not sure what was your initial intention

Comment (bodhinandach) [reply applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py:370]: That was also my question. But apparently the default parameter inside the function is designed to take:
```
"Parameters": {
        "materials_filename": "materials.json"
    }
```
not
`"materials_filename": "materials.json"`
So I followed the one in StructuralMechanics

Comment (philbucher) [reply applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py:370]: hmm strange
@loumalouomega @marandra why did you change the input-syntax in the c-utility?

Comment (bodhinandach) [reply applications/ParticleMechanicsApplication/python_scripts/particle_mpm_solver.py:370]: I will just merge this at the moment. Please kindly notify me if you think to modify the syntax in the future.

**Pull request merged**